contrib/ansible/openshift/README.md

```diff
@@ -0,0 +1,40 @@
+## Pbench Ansible
+Playbook to register pbench tools on openshift cluster
+
+### Requirements
+You need to have these installed
+   - Openshift
+   - pbench
+
+### Tools registered
+master - sar, pidstat, iostat, oc, pprof
+nodes - sar, pidstat, iostat, pprof
+etcd - sar, pidstat, iostat
+lb - sar, pidstat, iostat
+
+### Sample inventory
+Your inventory should have groups of hosts describing the roles they play in the openshift cluster. This playbook looks for the following groups in the inventory file:
+
+[pbench-controller]
+
+[masters]
+
+[nodes]
+
+[etcd]
+
+[lb]
+
+### Run
+Currently for pbench is run under the root user, so the playbook also needs to run as root.
+```
+$ ansible-playbook -i /path/to/inventory --extra-vars '{ "ansible_ssh_private_key_file":"" }' pbench_register.yml
+```
+By default the interval for ose_master_interval, ose_node_interval and default_tools_interval are set to 10 seconds.
+
+You can override the variables like
+```
+$ ansible-playbook -i /path/to/inventory --extra-vars '{"default_tools_interval":"30"}' pbench_register.yml
+```
+### labeling of nodes
+Each node has a label, index associated with it. Masters are labeled with svt_master_index, so for the first master, the label looks like svt_master_1. Similarly nodes are labeled with svt_node_index, etcd with svt_etcd_1 and lb with svt_lb_1. If the nodes in groups are same, for example master and etcd nodes are same then the label looks like svt_master_index_etcd_index.
```

Comment:
Can you change "_index" when referencing the abstract index number to be something like, "_<index>", so that folks know it is not the string "index" for clarity?